src/js/snp_attestation.cpp

```diff
@@ -0,0 +1,274 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+
+#include "ccf/js_plugin.h"
+#include "ccf/js_snp_attestation_plugin.h"
+#include "ccf/pal/attestation.h"
+#include "ccf/version.h"
+#include "js/wrap.h"
+
+#include <algorithm>
+#include <quickjs/quickjs.h>
+#include <regex>
+#include <vector>
+
+namespace ccf::js
+{
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wc99-extensions"
```

Comment:
Why is this required?

Reply:
https://github.com/microsoft/CCF/pull/5653/commits/b316adc7febba6ca788f11dc4b1771c6c3efde1f